### PR TITLE
refactor: ziofy ProjectsResponderV1 (DEV-1746)

### PR DESCRIPTION
**Pull request by seakayone**
### Task Description/Number

Issue Number: DEV-1746

### Basic Requirements

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### PR Type

What kind of change does this PR introduce?

- [ ] fix: represents bug fixes
- [x] refactor: represents production code refactoring
- [ ] feat: represents a new feature
- [ ] docs: documentation changes (no production code change)
- [ ] chore: maintenance tasks (no production code change)
- [ ] test: all about tests: adding, refactoring tests (no production code change)
- [ ] other... Please describe:

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No
- [ ] Maybe (not 100% sure => check with FE)

### Does this PR change client-test-data?

- [ ] Yes (don't forget to update the JS-LIB team about the change)
- [x] No

### CODE REVIEW

File: webapi/src/main/scala/org/knora/webapi/responders/v1/ProjectsResponderV1.scala [hunk 1]
```diff
@@ -477,17 +443,28 @@ final case class ProjectsResponderV1(actorDeps: ActorDeps) extends Responder(act
...
-
-      checkProjectExistsResponse <- appActor.ask(SparqlAskRequest(askString)).mapTo[SparqlAskResponse]
+      checkProjectExistsResponse <- triplestoreService.sparqlHttpAsk(askString)
       result                      = checkProjectExistsResponse.result
 
     } yield result
 }
+object ProjectsResponderV1Live {
+  val layer: URLayer[
+    StringFormatter with TriplestoreService with MessageRelay with IriService with AppConfig,
```
Comment (irinaschubert) [inline webapi/src/main/scala/org/knora/webapi/responders/v1/ProjectsResponderV1.scala:460]:
```suggestion
    StringFormatter with TriplestoreService with MessageRelay,
```

Comment (irinaschubert) [reply webapi/src/main/scala/org/knora/webapi/responders/v1/ProjectsResponderV1.scala:460]: these could probably be removed as well?

**Review by mpro7 — APPROVED** [review]
Please add the task number at the end of the PR title **in parenthesis** :)

**Review by irinaschubert — APPROVED** [review]
Just a minor detail but should probably be adjusted. Apart from that it looks good.

File: webapi/src/main/scala/org/knora/webapi/responders/v1/ProjectsResponderV1.scala (hunk 1 above)
Comment (seakayone) [reply webapi/src/main/scala/org/knora/webapi/responders/v1/ProjectsResponderV1.scala]: Thanks.

**Pull request merged**